pandas/tests/copy_view/test_interp_fillna.py

```diff
@@ -344,3 +346,16 @@ def test_fillna_inplace_ea_noop_shares_memory(
         assert not np.shares_memory(get_array(df, "b"), get_array(view, "b"))
     df.iloc[0, 1] = 100
     tm.assert_frame_equal(df_orig, view)
+
+
+def test_fillna_chained_assignment(using_copy_on_write):
+    df = DataFrame({"a": [1, np.nan, 2], "b": 1})
+    df_orig = df.copy()
+    if using_copy_on_write:
+        with tm.assert_produces_warning(ChainedAssignmentError):
```

Comment:
There is a helper for this, `with tm.raises_chained_assignment_error():`, which will take care of the fact that it doesn't raise a warning on PyPy

Reply:
thx